.github/workflows/static-analysis.yml

```diff
@@ -213,7 +213,13 @@ jobs:
         if: ${{ github.event_name != 'release' && github.event_name != 'schedule' }}
         run: npm run playwright install chromium --with-deps
 
-      - name: run unit tests for kcl samples
+      - name: Download internal KCL samples
+        run: git clone --depth=1 https://x-access-token:${{ secrets.GH_PAT_KCL_SAMPLES_INTERNAL }}@github.com/KittyCAD/kcl-samples-internal public/kcl-samples/internal
+
+      - name: Regenerate KCL samples manifest
+        run: cd rust/kcl-lib && EXPECTORATE=overwrite cargo test generate_manifest
```

Comment:
I don't love that this adds a couple minutes to the job, but it's nowhere near the slowest workflow. Ultimately, I think we need to completely restructure workflows based on shared dependencies.